packages/garbo/src/familiar/constantValueFamiliars.ts

```diff
@@ -152,3 +157,34 @@ export default function getConstantValueFamiliars(
       limit: "none",
     }));
 }
+
+const locationsWithMonsters = Location.all().filter(
+  (l) => getMonsters(l).length > 0,
+);
+
+function cookbookbatPerilBonus(): number {
+  if (!have($item`Peridot of Peril`)) return 0;
+  // canAdventure includes some zones we need to exclude
+  const canAdvExclusions = $locations`Fastest Adventurer Contest, Strongest Adventurer Contest, Smartest Adventurer Contest, Smoothest Adventurer Contest, Hottest Adventurer Contest, Coldest Adventurer Contest, Spookiest Adventurer Contest, Stinkiest Adventurer Contest, Sleaziest Adventurer Contest, The Hedge Maze, Tower Level 1, Tower Level 2, Tower Level 3, Tower Level 5, The Naughty Sorceress' Chamber, The Daily Dungeon, An Overgrown Shrine (Northwest), An Overgrown Shrine (Southwest), An Overgrown Shrine (Northeast), An Overgrown Shrine (Southeast), A Crater Full of Space Beasts, Mt. Molehill, The Red Queen's Garden, An Incredibly Strange Place (Bad Trip), An Incredibly Strange Place (Mediocre Trip), An Incredibly Strange Place (Great Trip), The Primordial Soup, The Jungles of Ancient Loathing, Seaside Megalopolis, Domed City of Ronaldus, Domed City of Grimacia, Hamburglaris Shield Generator, The X-32-F Combat Training Snowman, The Haiku Dungeon, The Deep Machine Tunnels, The Oasis, Shadow Rift, `;
```

Comment:
this might be worth making a top level `Set` instead of an array, depending upon how frequently this function gets called. I guess probably just once per turn?